src/display/annotation_layer.js

```diff
@@ -1926,14 +1925,8 @@ class ChoiceWidgetAnnotationElement extends WidgetAnnotationElement {
 
 class PopupAnnotationElement extends AnnotationElement {
   constructor(parameters) {
-    const { data, elements } = parameters;
-    const isRenderable = !!(
-      data.titleObj?.str ||
-      data.contentsObj?.str ||
-      data.richText?.str
-    );
-    super(parameters, { isRenderable });
-    this.elements = elements;
+    super(parameters, { isRenderable: true });
```

Comment:
Is this actually correct, since PopupAnnotations can be created in two different ways?
Either indirectly using the `_createPopup` helper-method for certain types of Annotations, where this modified `isRenderable` property should be OK.
However in PDF documents where PopupAnnotations are explicitly specified in the /Annots-array, don't we still need to keep a "proper" `isRenderable` property here?

Basically, I'd expect something like the following to address the second case (which ought to work fine for the first case as well):
```suggestion
    super(parameters, { isRenderable: this.hasPopupData });
```